deps/third/safetyhook/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include(FetchContent)
+
+set(FETCHCONTENT_QUIET OFF)
+
+FetchContent_Declare(
+    safetyhook
+    GIT_REPOSITORY "https://github.com/cursey/safetyhook.git"
```

Comment:
Can we switch to https://github.com/UE4SS-RE/safetyhook until our PR gets merged ?